apps/src/code-studio/assets/searchAssets.js

```diff
@@ -26,7 +26,7 @@ export function searchAssets(
   const searchRegExp = new RegExp('(?:\\s+|_|^|-)' + searchQuery, 'i');
 
   // Generate the set of all results associated with all matched aliases
-  let resultSet = Object.keys(assetLibrary.aliases)
+  let resultSet = Object.keys(assetLibrary.aliases || {})
```

Comment:
if tests (or a user) clicks too fast to load the add animation modal, before assetLibrary is loaded, you get a crash here that keeps the modal from appearing, this way (in the click very fast condition / tests) the modal appears and then populates a second or two later